### PR TITLE
Update pelephysics submodule

### PR DESCRIPTION
**Pull request by marchdf**
Changes appear in https://github.com/AMReX-Combustion/PelePhysics/pull/452

### CODE REVIEW

File: Tests/CMakeLists.txt [hunk 1]
```diff
@@ -45,7 +45,7 @@ macro(setup_test)
     set(PLOT_TEST ${CURRENT_TEST_BINARY_DIR}/plt00010)
     # Find fcompare
     if(PELEC_ENABLE_FCOMPARE_FOR_TESTS)
-      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/AMReX/Tools/Plotfile/fcompare)
+      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/PelePhysics/amrex/Tools/Plotfile/fcompare)
```
Comment (baperry2) [inline Tests/CMakeLists.txt]: Shouldn't this be `Submodules/PelePhysics/Submodules/amrex`?

Comment (marchdf) [reply Tests/CMakeLists.txt]: yes! good catch. It explains some of the failures I was seeing on the CI. Thank you.

**Review by baperry2 — APPROVED** [review]

File: Tests/CMakeLists.txt [hunk 2]
```diff
@@ -45,7 +45,7 @@ macro(setup_test)
     set(PLOT_TEST ${CURRENT_TEST_BINARY_DIR}/plt00010)
     # Find fcompare
     if(PELEC_ENABLE_FCOMPARE_FOR_TESTS)
-      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/AMReX/Tools/Plotfile/fcompare)
+      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/PelePhysics/Submodules/amrex/Tools/Plotfile/fcompare)
```
Comment (jrood-nrel) [inline Tests/CMakeLists.txt]: This should have the renamed `fcompare`.

**Review by jrood-nrel — APPROVED** [review]

File: Tests/CMakeLists.txt (hunk 2 above)
Comment (marchdf) [reply Tests/CMakeLists.txt]: thanks, missed that one.

**Comment by baperry2** [comment]
Not what was causing the clang-tidy problems, but we should fix the filename filter so it's not capturing all the amrex and sundials stuff again, like here: https://github.com/AMReX-Combustion/PeleLMeX/pull/325/commits/ce16b49c313ea173b1ebc979f3bd3d8f8f5e7ea7

**Pull request merged**